compiler/codegen/expr.cpp

```diff
@@ -453,7 +453,8 @@ llvm::StoreInst* codegenStoreLLVM(llvm::Value* val,
   GenInfo *info = gGenInfo;
   llvm::StoreInst* ret = info->builder->CreateStore(val, ptr);
   llvm::MDNode* tbaa = NULL;
-  if( USE_TBAA && valType ) tbaa = valType->symbol->llvmTbaaAccessTag;
+  if( USE_TBAA && valType && !valType->symbol->llvmTbaaStructCopyNode )
+    tbaa = valType->symbol->llvmTbaaAccessTag;
```

Comment:
It seems surprising to me that we won't mark a store or a load with TBAA metadata if a StructCopyNode is set for that type.

Reply:
This is to prevent the wrong kind of TBAA node from being used.  More is coming.